dataplane/apigen/ccgen/ccgen.go

```diff
@@ -290,6 +298,44 @@ func createCCData(meta *saiast.FuncMetadata, apiName string, sai *saiast.SAIAPI,
 		convertFn = nil
 	}
 
+	// Patches for non-standard APIS
+	if meta.TypeName == "ACL_TABLE" {
+		switch meta.Operation {
+		case createOp:
+			convertFn.AttrConvertInsert = `
+if (attr_list[i].id >= SAI_ACL_TABLE_ATTR_USER_DEFINED_FIELD_GROUP_MIN && attr_list[i].id < SAI_ACL_TABLE_ATTR_USER_DEFINED_FIELD_GROUP_MAX) {
```

Comment:
Do we need to care about the else situation?

Reply:
no, only need the special logic for these cases